### DISCUSSION TITLE
Run IOPaint inside Google Colab

### DISCUSSION THREAD

**Discussion by NotFenixio** [Sanster/IOPaint#445] [Show and tell]
Hey, I made a small script to run IOPaint in Google Colab and expose the port using Ngrok.
You just need to insert your Ngrok token, select your favorite model, and done.
[📒Notebook here](https://colab.research.google.com/drive/1wg3Y874eAgdaX0EIjIgTZROze3hqXyo7?usp=sharing)

**Comment by Sanster** [comment]
Thank you for creating this Notebook! I have added it to the README

**Comment by wolfkingal2000** [comment]
> Hey, I made a small script to run IOPaint in Google Colab and expose the port using Ngrok.
> You just need to insert your Ngrok token, select your favorite model, and done.
> [📒Notebook here](https://colab.research.google.com/drive/1wg3Y874eAgdaX0EIjIgTZROze3hqXyo7?usp=sharing)

That cool can you add controlnet and plugins too? Or how can add them 🤔

**Reply by NotFenixio** [reply]
I made this script to test out Ngrok, and decided to share it with the community
I don't think I can really implement anything since I'm not familiarized with IOPaint, but it would certainly be cool if someone updated the script to add support to these things.
